OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -517,7 +516,7 @@ public void ResolveOrder(Actor self, Order order)
 		{
 			if (order.OrderString == "Move")
 			{
-				if (!Info.MoveIntoShroud && !self.Owner.Shroud.IsExplored(order.TargetLocation))
+				if (IsTraitDisabled || (!Info.MoveIntoShroud && !self.Owner.Shroud.IsExplored(order.TargetLocation)))
```

Comment:
Same here as in the above comment. `if ((!order.Queued && IsTraitDisabled) || ...)`